### ISSUE TITLE
Support for Angular 6.0.0+

### ISSUE THREAD

**Issue by tomflenniken** [Teradata/covalent-code-editor#35]
My versions: 

```
    ...
    "@angular/common": "6.0.0",
    "@angular/core": "6.0.0",
    "@covalent/core": "2.0.0-beta.1",
    "@covalent/code-editor": "^1.0.0",
    ...
```

Installing gives a warning: 
```
npm WARN @covalent/code-editor@1.0.0 requires a peer of @angular/common@^5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @covalent/code-editor@1.0.0 requires a peer of @angular/core@^5.0.0 but none is installed. You must install peer dependencies yourself.
```

Running gives an error:
```
ERROR in node_modules/@covalent/code-editor/code-editor.component.d.ts(3,10): error TS2305: Module '"{my project directory}/node_modules/rxjs/Observable"' has no exported member 'Observable'.
node_modules/rxjs/Observable.d.ts(1,15): error TS2307: Cannot find module 'rxjs-compat/Observable'.
```

What I would like:

Compatibility for @angular versions 6.0.0 and up and @covalent versions 2.0.0 and up.

**Comment by jeremysmartt** [comment]
thanks for the issue, will take a look shortly. 👍

**Comment by jeremysmartt** [comment]
@tomflenniken Can you try out this version of the Code Editor:
https://www.npmjs.com/package/@covalent/code-editor/v/2.0.0-beta.1

It should work for you in Angular 6

**Comment by tomflenniken** [comment]
Oh thanks I didn't realize this version was available.

**Comment by altoic** [comment]
If the problem comes up again I found that doing 

`npm install --save rxjs-compat` 

will fix the issue on 6 while still using v1.0.0 of the code editor

### my env
```
Angular CLI: 6.0.0
Node: 10.0.0
OS: win32 x64
Angular: 6.0.0
```
